tests/Stripe/SourceTest.php

```diff
@@ -121,6 +121,18 @@ public function testCanListSourceTransactions()
         $this->assertInstanceOf(\Stripe\SourceTransaction::class, $resources->data[0]);
     }
 
+    public function testCanListSourceTransactions()
+    {
+        $source_id = self::TEST_RESOURCE_ID;
```

Comment:
Super nit, but can you remove this variable and just use the class constant like you did in the other test?

Reply:
heh, yep! wrote this one first, being lazy